docs/handling-touches.md

```diff
@@ -26,11 +26,11 @@ Go ahead and play around with the `Button` component using the example below. Yo
 
 ```SnackPlayer name=Button%20Basics
 import React, {Component} from 'react';
-import {Button, StyleSheet, View} from 'react-native';
+import {Alert, Button, StyleSheet, View} from 'react-native';
```

Comment:
RN's current ESLint config enables `no-alert`, which bans the raw `alert()` API. But using the RN `Alert` API is seen as fine.